packages/foreman/foreman-selinux/foreman-selinux.spec

```
@@ -36,16 +36,28 @@ License:        GPLv3+
 URL:            https://theforeman.org
 Source0:        https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerelease:-}%{?prerelease}.tar.bz2
 
-BuildRequires:  checkpolicy, selinux-policy-devel, hardlink
+BuildRequires:  checkpolicy
+BuildRequires:  selinux-policy-devel
+BuildRequires:  hardlink
 BuildRequires:  policycoreutils
 BuildRequires:  /usr/bin/pod2man
 BuildArch:      noarch
 
 Requires:           selinux-policy >= %{selinux_policy_ver}
-Requires(post):     /usr/sbin/semodule, /sbin/restorecon, /usr/sbin/setsebool, /usr/sbin/selinuxenabled, /usr/sbin/semanage
-Requires(post):     policycoreutils-python
+Requires(post):     /usr/sbin/semodule
+Requires(post):     /sbin/restorecon
+Requires(post):     /usr/sbin/setsebool
+Requires(post):     /usr/sbin/selinuxenabled
+Requires(post):     /usr/sbin/semanage
 Requires(post):     selinux-policy-targeted
-Requires(postun):   /usr/sbin/semodule, /sbin/restorecon
+Requires(postun):   /usr/sbin/semodule
+Requires(postun):   /sbin/restorecon
+
+%if 0%{?rhel} < 8
```

Isn't this still resulting in using `policycoreutils-python` on Fedora since that doesn't define `%{rhel}`
```suggestion
%if 0%{?rhel} == 7
```